Makefile

```diff
@@ -11,4 +11,4 @@ start-dev: env-setup
 	forego start
 
 test: env-setup
-	go test -v ./...
+	go test -v -p 1 ./...
```

Comment:
What do the extra params `-p 1` do?

Reply:
According to `go build help`
```
-p n
		the number of programs, such as build commands or
		test binaries, that can be run in parallel.
		The default is the number of CPUs available.
```
I found that run test failed sometimes because there are more record in the database than I expected but sometimes not.

Reply:
Looks like some race conditions. That fix is more like a band-aid 🩹 as basically it disables running tests in parallel. So it might have to do with the test set up 🤔 In controller tests for instance since the `SetupTest()` is shared for all tests and each test modifies the test data, there is no real isolation between each test. So I belive this is an area to research more on.

Reply:
Noted